joint_trajectory_controller/test/test_trajectory_controller.cpp

```diff
@@ -644,7 +646,7 @@ TEST_P(TrajectoryControllerTestParameterized, position_error_not_angle_wraparoun
   size_t n_joints = joint_names_.size();
 
   // send msg
-  constexpr auto FIRST_POINT_TIME = std::chrono::milliseconds(250);
+  constexpr auto FIRST_POINT_TIME = std::chrono::milliseconds(300);
```

Comment:
If the control period is 100 milliseconds, shouldn't it be 350?
Am i missing something here?

Reply:
Why? Would it be desired to set a target time between two control cycles?

With a control period of 100 ms we would call `update` at $t_0 + 100ms, t_0 + 200ms, t_0+300ms$... So, in the update call at $t=t_0+200ms$ it should generate the command to reach the setpoint which is what we wait for below.